src/main/java/seedu/address/model/person/Person.java

```diff
@@ -52,6 +69,10 @@ public Address getAddress() {
         return address;
     }
 
+    public seedu.address.model.person.Gender getGender() {
```

Comment:
Any reason why the `Gender` class is referenced directly here instead of being imported at the top of the file?